patches/tModLoader/Terraria/NetMessage.cs.patch

```diff
@@ -372,6 +377,15 @@
  							short num11 = (short)Sign.ReadSign(j, i);
  							if (num11 != -1)
  								array2[num2++] = num11;
+@@ -1738,7 +_,7 @@
+ 								array3[num3++] = (short)num18;
+ 						}
+ 
+-						if (tile2.type == 597 && tile2.frameX % 54 == 0 && tile2.frameY % 72 == 0) {
++						if (TileID.Sets.CountsAsPylon.Contains(tile2.type) && tile2.frameX % 54 == 0 && tile2.frameY % 72 == 0) {
```

Comment:
If you use a modded pylon TE for all pylons then some of these `CountsAsPylon.Contains` patches should disappear